libraries/lib-string-utils/FromChars.h

```diff
@@ -0,0 +1,52 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file FromChars.h
+ @brief Declare functions to convert numeric types to string representation.
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#pragma once
+
+#include <system_error>
+
+//! Result of the conversion, similar to std::from_chars_result
+struct STRING_UTILS_API FromCharsResult final
+{
+   const char* ptr; //! A pointer to the first character no matching the the pattern
+
+   //! std::errc::invalid_argument, if there is no pattern mactch,
+   //! std::errc::result_out_of_range if the value parsed is to large,
+   //! std::errc() on success
+   std::errc ec; 
+};
+
+//! Parse a string into a single precision floating point value, always uses the dot as decimal
+STRING_UTILS_API FromCharsResult FromChars(const char* buffer, const char* last, float& value) noexcept;
+
+//! Parse a string into a single precision floating point value, always uses the dot as decimal
+STRING_UTILS_API FromCharsResult FromChars(const char* buffer, const char* last, double& value) noexcept;
+
+//! Parse a string into a singed 16-bit integer value
```

Comment:
"signed" (in 4 places)

"singed" means "burned" -- which you could also say as "charred" :-)

Reply:
(And it rhymes with "cringed")